ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/CommandDeduplicationIT.scala

```diff
@@ -12,23 +12,22 @@ import com.daml.ledger.test.model.DA.Types.Tuple2
 import com.daml.ledger.test.model.Test.TextKeyOperations._
 import com.daml.ledger.test.model.Test._
 import com.daml.timer.Delayed
-import com.google.protobuf.duration.Duration
 import io.grpc.Status
 
 import scala.concurrent.Future
-import scala.concurrent.duration.DurationInt
 import scala.util.{Failure, Success}
 
-final class CommandDeduplicationIT extends LedgerTestSuite {
+final class CommandDeduplicationIT(ledgerTimeIntervalMs: scala.concurrent.duration.Duration) extends LedgerTestSuite {
```

Comment:
The duration encapsulates the time unit.

```suggestion
final class CommandDeduplicationIT(ledgerTimeInterval: scala.concurrent.duration.Duration) extends LedgerTestSuite {
```

Reply:
addressed